numba/cuda/dispatcher.py

```diff
@@ -96,6 +106,17 @@ def __init__(self, py_func, argtypes, link=None, debug=False,
         if self.cooperative:
             link.append(get_cudalib('cudadevrt', static=True))
 
+        res = [ele for ele in cuda_fp16_math_funcs
```

Comment:
`ele` is a strange choice of name for the loop variable here. Perhaps `f` or `fn` are a closer representation of what the variable holds?